services/frontend/src/pages/tasks.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import TasksPage from '../components/tasks/tasks-page';
 
-export default function wrapper() {
-    return <TasksPage />;
+export default function Tasks() {
+    return <TasksPage></TasksPage>;
```

Comment:
Bad change from the merge?

Reply:
This is a change that eslint requested or the test would fail

Reply:
Are you sure? It works on master just fine

Reply:
im sure, the test results on github pointed directly to this line, I'll try to revert it

Reply:
it seemed to have worked